tests/test_sft_trainer.py

```diff
@@ -29,7 +29,9 @@
 import transformers
 
 # First Party
+from build.utils import serialize_args
 from scripts.run_inference import TunedCausalLM
+from tests.build.test_launch_script import BASE_LORA_KWARGS
 from tests.data import (
     EMPTY_DATA,
     MALFORMATTED_DATA,
```

Comment:
Would be good to add a test where run parse_arguments and verify that not passing in target_modules gives None still. This can be added to [existing test on line 149](https://github.com/foundation-model-stack/fms-hf-tuning/pull/281/files#diff-406c3c4d02c1ff158009b04d26a6b2d357f05d11e8c35c3f2ddfc1b54649a022R149)